tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -81,31 +92,45 @@ def ValidateMessage(self, message):
 
     if entity_name not in self.entities:
       self.AddError(
-        telemetry_error.TelemetryError(
-          entity_name, None, "Unknown entity"))
+          telemetry_error.TelemetryError(
+              entity_name, None, "Unknown entity"))
       message.ack()
       return
 
     if entity_name in self.validated_entities:
       # Already validated telemetry for this entity,
       # so the message can be skipped.
+      message.ack()
       return
     self.validated_entities[entity_name] = True
 
     entity = self.entities[entity_name]
+    print("Validating telemetry message for entity: {0}".format(entity_name))
+
+    if TRANSLATION not in entity.keys():
+      # TODO: ignore if the type is a facility
+      print("Missing Translation in Building Config for entity: {0}"
+            .format(entity_name))
```

Comment:
Probably better to just add the warning, following the general principle of not logging + throwing exceptions.

We can still (optionally) print out the warning message in the report.